connectors/web/src/main/java/org/crsh/web/servlet/WSProcessContext.java

```diff
@@ -147,10 +147,10 @@ public Appendable append(CharSequence csq, int start, int end) throws IOExceptio
       } else {
         Style.Composite composite = (Style.Composite)style;
         buffer.append("[[");
-        if (composite.getUnderline() == Boolean.TRUE) {
+        if (Boolean.TRUE.equals(composite.getUnderline())) {
```

Comment:
These are probably valid.  Is this code supposed to check the *identity* of the booleans, or their value?  It might be syntactically nicer to test like "if (composite.getUnderline())" would be even better, because it is simpler.  This comment applies for all of the boolean comparisons in this file.